eng/pipelines/mono/templates/xplat-job.yml

```diff
@@ -90,6 +90,16 @@ jobs:
     - name: osSubgroup
       value: ${{ parameters.osSubgroup }}
 
+    - name: coreClrRepoRoot
+      value: '$(Build.SourcesDirectory)/src/coreclr'
+
+    - name: coreClrRepoRootDir
+      value: '$(coreClrRepoRoot)$(dir)'
+
+    - name: priorityArg
+      value: ''
```

Comment:
Is this always needed? I don't see this being set.

Reply:
Removed it.